test/boost/tablets_test.cc

```diff
@@ -109,6 +105,37 @@ future<table_id> add_table(cql_test_env& e, sstring test_ks_name = "") {
     co_return id;
 }
 
+// Run in a seastar thread
+static
+sstring add_keyspace(cql_test_env& e, std::unordered_map<sstring, int> dc_rf, int initial_tablets = 0) {
```

Comment:
maybe initial_tablets should be optional, in case the test wants to create a keyspace with no `'initial':X` option?

Reply:
`'initial' = 0` in keyspace options has the same effect as no `initial`.